test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua

```diff
@@ -0,0 +1,141 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary: 
+--    [Policies]: lack of "user_consent_prompt" field is included to the <appID>`s policies
+--
+-- Description: 
+--     PTU for <appID> contains policies with <functional grouping> that lacks "user_consent_prompt" field
+--     1. Used preconditions:	
+--			Unregister default app
+--			Register application
+--			Consent device-> Activate application
+--
+--     2. Performed steps
+--			Perform PTU that contains policies for app with <functional grouping> that lacks "user_consent_prompt" field		   
```

Comment:
rejected, on PTU step also is checked that param "appPermissionsConsentNeeded" is "false"